### PR TITLE
Added the float Action button in Homepage

### PR DESCRIPTION
**Pull request by samrath-sudesh-acharya**
Screenshot_1645429599

### CODE REVIEW

**Comment by samrath-sudesh-acharya** [comment]
:)
close #193

**Comment by hashirshoaeb** [comment]
Thanks for showing your interest. I'll test it later today. :) 🚀 ❤️

File: lib/screens/home_page.dart [hunk 1]
```diff
@@ -162,6 +168,17 @@ class _HomePageState extends State<HomePage> {
                     ),
                   ),
                 ),
+                c.Padding(
```
Comment (hashirshoaeb) [inline lib/screens/home_page.dart:171]: Use Material based `Scaffold`, to position Floating button

Comment (samrath-sudesh-acharya) [reply lib/screens/home_page.dart:171]: When I used that there was render overflow

Comment (samrath-sudesh-acharya) [reply lib/screens/home_page.dart:171]: ok i will do that

**Comment by samrath-sudesh-acharya** [comment]
@hashirshoaeb i have a problem , when i use the endFloat location for float bouttom in scaffold , i get overflow ...
To you know any way to solve the problem

**Comment by hashirshoaeb** [comment]
> @hashirshoaeb i have a problem , when i use the endFloat location for float bouttom in scaffold , i get overflow ... To you know any way to solve the problem

Hmm, Thanks for sharing. Let me look into this. I'll get back to you. meanwhile I'll encourage you to work on other issues.

**Comment by hashirshoaeb** [comment]
Pushing a fix.

**Comment by hashirshoaeb** [comment]
learned from here: https://stackoverflow.com/questions/58077794/can-i-dock-a-floating-action-button-in-a-cupertinotabbar

**Pull request merged**